diff --git a/dist/cjs/slim.cjs b/dist/cjs/slim.cjs
index 21ce43a..996bd71 100644
--- a/dist/cjs/slim.cjs
+++ b/dist/cjs/slim.cjs
@@ -27,5 +27,6 @@ __export(slim_exports, {
   RawString: () => RawString,
   Uint: () => Uint,
+  UseApi: () => UseApi,
   addCommits: () => addCommits,
   addFragments: () => addFragments,
   applyChanges: () => applyChanges,
@@ -3929,5 +3930,6 @@
   RawString,
   Uint,
+  UseApi,
   addCommits,
   addFragments,
   applyChanges,
diff --git a/dist/index.d.ts b/dist/index.d.ts
index c74d30f..c56775d 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -229,2 +229,3 @@
 export * from "./implementation.js";
 export * as next from "./implementation.js";
+export { UseApi } from "./low_level.js";
diff --git a/dist/mjs/entrypoints/slim.js b/dist/mjs/entrypoints/slim.js
index e0e8464..a58ca8a 100644
--- a/dist/mjs/entrypoints/slim.js
+++ b/dist/mjs/entrypoints/slim.js
@@ -1 +1,2 @@
 export * from "../index.js";
+export { UseApi } from "../low_level.js";
