diff --git a/dist/module/metro/injection-code.js b/dist/module/metro/injection-code.js
index 669780d..4514abc 100644
--- a/dist/module/metro/injection-code.js
+++ b/dist/module/metro/injection-code.js
@@ -10,12 +10,15 @@
  *
  * To fix this, we force our code to always import the CSS files, so now the CSS files are in every bundle.
  */
+function importPathForMetro(filePath) {
+  return filePath.replace(/\\/g, "/");
+}
 export function getWebInjectionCode(filePaths) {
-  const importStatements = filePaths.map(filePath => `import "${filePath}";`).join("\n");
+  const importStatements = filePaths.map(filePath => `import "${importPathForMetro(filePath)}";`).join("\n");
   return Buffer.from(importStatements);
 }
 export function getNativeInjectionCode(cssFilePaths, values) {
-  const importStatements = cssFilePaths.map(filePath => `import "${filePath}";`).join("\n");
+  const importStatements = cssFilePaths.map(filePath => `import "${importPathForMetro(filePath)}";`).join("\n");
   const contents = values.map(value => `StyleCollection.inject(${JSON.stringify(value)});`).join("\n");
   return Buffer.from(`import { StyleCollection } from "react-native-css/native-internal";\n${importStatements}\n${contents};export {};`);
 }
